exercises/sgf-parsing/description.md

```diff
@@ -53,6 +53,13 @@ A key can have multiple values associated with it. For example:
 
 Here `AB` (add black) is used to add three black stones to the board.
 
+Being that parentheses, square brackets and semicolons are integral parts of
+the SGF format they must be escaped by preceding the character with a backslash
+(i.e. `\(`, `\;` and etc.) when found in text. Some other characters that
+require escaping are newlines, tabs and backslashes (`\n`, `\t`, `\\`). In
```

Comment:
These four lines are relevant to someone who wants to output a string in the SGF format. Students are not asked to do that (students are parsing, not outputting), so this information is more information for them to read which ultimately does not help them.

However, it is useful to discuss how to deal with escape sequences when parsing. So perhaps instead just a short bit like "characters preceded by a backslash (the escape character) appear in the output as-is, without the backslash" ?

Finally, if these four lines are instead kept as-is, I cannot verify their truthfulness, because they contradict what's said in https://www.red-bean.com/sgf/sgf4.html

> Escaping: "\" is the escape character. Any char following "\" is inserted verbatim (exception: whitespaces still have to be converted to space!). Following chars have to be escaped, when used in Text: "]", "\" and ":" (only if used in compose data type).

Reply:
1. I can fix the language to explain how/when escaped characters will be encountered instead of saying that chars need to be escaped.
2. I can update the content to conform to what the spec says
3. one thing I don't find clear in the spec is how to handle line-breaks, the expectation in this exercise is clearly to replace the C# string `"\\n"` with the char `\n` so I'm going to go with that for now

Reply:
updated, what do you think?